src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {useState, useContext} from "react";
+import { UserContext } from '../../context/userContext'
+
+import "./Profile.css";
+
+function Profile() {
+    const user = useContext(UserContext);
```

Comment:
Once you change to use path params, you will have to change this form using the logged in user to the id of the user thats passed on. You will have to make a  get user request to the server. I don't remember how to get path params but you will have to figure that out too. it might be `useParams()`